app/models/manager_refresh/save_collection/saver/base.rb

```diff
@@ -121,10 +121,10 @@ def save!(association)
             end
           end
         end
-        _log.info("*************** PROCESSED #{inventory_collection}, "\
-                  "created=#{inventory_collection.created_records.count}, "\
-                  "updated=#{inventory_collection.updated_records.count}, "\
-                  "deleted=#{inventory_collection.deleted_records.count} *************")
+        _log.debug("Processing #{inventory_collection}, "\
```

Comment:
I would keep at least this one on info? That should help us to see how many&how long, in a case of perf issues.

Reply:
which goes with (not enough logging for old refresh made it hard to pinpoint the source of issue)

Reply:
I think some of these are useful but not sure if they all are, for instance a single refresh of an openshift provider gave us 45 of just these `created/updated/deleted` lines.  I think these are great for top level objects (Container, ContainerGroup, ContainerImage) but maybe not ContainerEnvVar, CustomAttribute? @Fryguy WDYT?

> in a case of perf issues.

This is when we'd turn debug on and re-collect the logs IMO

Reply:
If it's ok to expect that people will run debug first, then I am fine with it. :-) I just know it was biting us a lot, that old refresh has no logging for figuring out where is the perf problem.